ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -0,0 +1,135 @@
+"""
+Any fixtures involving loading data from the bucket.
+"""
+
+import json
+from typing import Dict, List
+
+import mdocfile
+import ndjson
+import pandas as pd
+import pytest
+from data_validation_common.mrc import get_header
+from mrcfile.mrcinterpreter import MrcInterpreter
+
+from common.fs import FileSystemApi
+
+
+### Tilt series fixtures ###
+@pytest.fixture(scope="session")
+def tiltseries_mrc_headers(tiltseries_mrc_files: List[str]) -> Dict[str, MrcInterpreter]:
+    """Get the mrc file headers for a tilt series."""
+    headers = {}
+    for file in tiltseries_mrc_files:
+        try:
+            headers[file] = get_header(file)
+        except Exception as _:
+            pytest.fail(f"Failed to get header for {file}")
+
+    return headers
+
+
+@pytest.fixture(scope="session")
+def tiltseries_metadata(tiltseries_meta_file: str, filesystem: FileSystemApi) -> Dict:
+    """Load the tiltseries metadata."""
+    with filesystem.open(tiltseries_meta_file, "r") as f:
+        return json.load(f)
+
+
+@pytest.fixture()
+def tiltseries_mdoc(tiltseries_mdoc_file: str, filesystem: FileSystemApi) -> pd.DataFrame:
+    """Load the tiltseries mdoc."""
+
+    with filesystem.open(tiltseries_mdoc_file, "r") as f:
+        return mdocfile.read(f)
+
+
+@pytest.fixture()
+def tiltseries_tlt(tiltseries_tlt_file: str, filesystem: FileSystemApi) -> pd.DataFrame:
+    """Load the tiltseries tlt."""
+
+    with filesystem.open(tiltseries_tlt_file, "r") as f:
+        return pd.read_csv(f, sep=r"\s+", header=None, names=["tilt_angle"])
+
+
+### Tomogram fixtures ###
+@pytest.fixture(scope="session")
+def canonical_tomo_mrc_headers(canonical_tomo_mrc_files: List[str]) -> Dict[str, MrcInterpreter]:
+    """Get the mrc file headers for a tomogram."""
+    headers = {}
+    for file in canonical_tomo_mrc_files:
+        try:
+            headers[file] = get_header(file)
+        except Exception as _:
+            pytest.fail(f"Failed to get header for {file}")
+
+    return headers
+
+
+@pytest.fixture(scope="session")
+def canonical_tomogram_metadata(canonical_tomo_meta_file: str, filesystem: FileSystemApi) -> Dict:
+    """Load the canonical tomogram metadata."""
+    with filesystem.open(canonical_tomo_meta_file, "r") as f:
+        return json.load(f)
+
+
+### Annotation fixtures ###
+@pytest.fixture(scope="session")
+def annotation_metadata(annotation_meta_files: str, filesystem: FileSystemApi) -> Dict[str, Dict]:
```

Comment:
Should `annotation_meta_files` be a list[str]?

Reply:
fixed!